doc/icub-whole-body-torque-control-faqs.md

```diff
@@ -6,3 +6,102 @@ Frequent Asked Questions on whole-body torque control on the iCub robot  {#icub-
+There are BLL and RMHDC. For the two one download the corresponding file.out.s (that correspond to the actual version).  
+
+# Calibration of the robot
+Do the calibration in this order: from torso to the legs, then the head and the arm.  
+You can follow [these informations](http://wiki.) to do it this [part](http://wiki.icub.org/) explain quickly what you have to do.  
+The following lines gives other information and an example.  
+In _Yarpmanager_, put the member you want to calibrate move in the “idle” mode to allow you to move it. With a level, put the joint in the parallel position to the floor. Then put the mode of the considered joint in “run mode”. Note the encoder value.  
+This value has to be added in the specific file linked to the joint. Go to the folder:  
+>>$CODYCO_SUPERBUILD_ROOT/external/ICUB/app/robots/$ROBOT_NAME/conf/calibration  
+** /*Check it*/ **  
+Search the corresponding file (for example left_leg_calib.xml), and add the “encoder value” that you have noted to the actual one (parameter _calibrationDelta_: all numbers correspond to the joints). For example, we add some values to the line:  
+> <param name="calibrationDelta">      
+
+Thus, the source file shared between the load_computer and the icub_board (pc104) has been changed. Now, you have to update the icub_board. 
+> ssh -X pc104
+> cd $YARP_DIR/icub_main/build
+> ccmake […]
+> make  
+
+At the end, if all is correct, you have to commit it.  
+_Remarks_:  
+It is better to do many commit and not to commit at the end all the configuration files to avoid errors.  
+If you have the same configuration file in the local folder, the computer will use this file and not the one you modify. In that case, when you do test, modify the file in the local folder. When you are sure of the calibration, modify the correct one and commit it.
+  
+To calibrate the head:  
+The configuration file for the head is on another folder:
+> $CODYCO_SUPERBUILD_ROOT/
+Take the file linked to the head and modify the _zero parameter_.
+You need to receive the acceleration values of the head to be able to calibrate it: launch a program who gives these data. Like previously, choose to the _idle mode_ in the _yarpmanager_ for the two first joints of the head. Then, turn the head until the acceleration values are near to (0;0;9) (the gravity as to be sensed only on the z axis). If x/y position are less than 0.1 it is ok.  After that, click on run and copy the encoder value on the corresponding XML file. 
+  
+To calibrate the arm:  
+Follow [this configuration](http://wiki.).
+
+
+### **I also propose to add this following part on the installation guide... But I understand if you think it is too much: the installation guide is already long so maybe it is better to don't right that. I can also create an issue with that, where I could write something like “seems I have had some trouble during the installation, I think this information can be useful to avoid them”.**
```

Comment:
Good idea. Given that as you told the installation guide is quite long, we can move this part to an installation tips and tricks page.